kqueen/models.py

```diff
@@ -62,6 +64,13 @@ def get_state(self):
 
         return self.state
 
+    def set_status(self, cluster):
+        if cluster:
```

Comment:
What do you expect to get that will not pass this condition? Manually passed None or empty string?

Reply:
in case of fail cluster is empty

Reply:
Empty dict? If so, you don't have to check it here, because you check it below